.gitignore

```diff
@@ -20,3 +20,10 @@ packages
 pascal/lib/
 pascal/backup/
 .vs
+
+# pks
+3rdparty
+Backup/
+WinSparkle.vcxproj
+*.filters
+examples/example_psdk.vcxproj
```

Comment:
This does not ignore "build artifacts", but adds, wrongly, many files *that are committed to the repository* (which never makes any sense), ignores requires submodules (ditto) and adds some private files of yours (use your local git config for that).

Reply:
Again, this wasn't meant for your repo, sorry.